site/components/Navigation/index.tsx

```diff
@@ -58,47 +64,345 @@ export const Navigation: FC<Props> = ({
         activePage={activePage}
       >
         <NavItemDesktop
-          url={createLinkWithLocaleQuery(urls.offset, locale)}
-          name={t({ message: "Offset", id: "shared.offset" })}
+          name={t({ message: "About", id: "shared.about" })}
+          subMenu={[
+            <NavItemDesktop
+              name={t({
+                message: "BLOG",
+                id: "shared.blog",
+              })}
+              key="blog"
+              href="/blog"
```

Comment:
`NavItemDesktop` does not have the prop `href` 
Instead the href is passed as `url`, otherwise none of the Links are working right now.

Reply:
fixed